README.adoc

```diff
@@ -338,3 +338,24 @@ Where :
 
 - SHA1: is the given git SHA1 of the james-project repository to merge.
 - RESULTING_BRANCH: is the branch created when merging.
+
+=== Generating a debian package for James
+
+You can generate a debian package for James by using the following process.
+
+First step, you have to build the Docker image
+
+    $ docker build -t build-james-deb --build-arg ITERATION=1 dockerfiles/packaging/debian/
+
+Where:
+
+- ITERATION is increment of the release of the package (e.g. 3.0-beta6-1, 3.0-beta6-2, 3.0-beta6-3...)
```

Comment:
That's it, has it has several names on packaging tools (and users may not know the deb therms) I thought that an example was a good choice.